src/features/ContentTypes.feature

```diff
@@ -0,0 +1,161 @@
+      And i click Discard changes
+    Then i shoud be redirected to Content Type Group context
+      And i should see Content Types list
+      And there's nothing new on the list
+
+  @javascript @common
+  Scenario: New Field can be added to new Content Type
+    Given that im in context of Content Type Group
+    When i click plus button
+      And i fill Name field
+      And i fill Identifier field
+      And i fill Content name pattern field
+      And i select Text line content field definition
+      And Click Add field definition
+    Then message "Content type 'content_type_name' updated." appears
+      And form for new fieldtype definition appears.
+
+  @javascript @common
+  Scenario: New Content Type can be added to Content Type Group
+    Given that i'm in context of content type update form
+      And there are all necessary field filled
+      And there is at least one field added
+    When i click Save
+    Then i should be redirected to new Content Type context
+      And Content Type has proper values
+      And Content Type has proper fields
+      And message "Content type 'content_type_name' updated." appears
+
+  @javascript @common
+  Scenario: I can navigate to Content Type Group through breadcrumb
+    Given that i'm in context of Content Type
+    When i click on Content Type group name on breadcrumb
+    Then i shoud be redirected to Content Type Group context
+      And i should see Content Types list
+
+  @javascript @common
+  Scenario: Changes can be discarded while editing Content type
+    Given that i'm in context of Content Type Group
+      And there's Content Type on the list
+    When i click Edit button left to 'content_type_name'
+      And fill Name field.
+      And click Discard changes
+    Then i shoud be redirected to Content Type Group context
+      And i should see Content Types list
+      And changes shoudn't be applied to content type name
+
+  @javascript @common
+  Scenario: New Field can be added while editing Content Type
+    Given that im in context of Content Type Group
+      And there's Content Type on the list
+    When i click Edit button left to 'content_type_name'
+      And i select Date content field definition
+      And Click Add field definition
+    Then message "Content type 'content_type_name' updated." appears
+      And form for new fieldtype definition appears.
+
+  @javascript @common
+  Scenario: Content type can be edited
+    Given that i'm in context of Content Type update form
+      And there's Content Type on the list
+      And i fill Name field.
+      And click Save
+    Then i should be redirected to new Content Type context
+      And Content Type has proper values
+      And Content Type has proper fields
+      And message "Content type 'content_type_name' updated." appears
+
+  @javascript @common
+  Scenario: Changes of Content Type are visible in Content Type Group context
+    Given that i'm in context of just edited Content Type
```

Comment:
Something to discuss: to what extent do we plan to allow Scenarios to rely on each other?
If I read it correctly, it depends on the previous Scenario (and maybe we could simply Edit another Content Type in this test).